server/src/main/java/com/objectcomputing/checkins/services/employee_hours/EmployeeHours.java

```diff
@@ -55,29 +57,28 @@ public class EmployeeHours {
     @Column(name="targethours")
     @NotNull
     @Schema(description = "Target hours for an employee", required = true)
-    private float targetHours;
+    private Float targetHours;
 
     @Column(name="asofdate")
     @Schema(description = "as of Date")
     private LocalDate asOfDate;
 
-
-
-    public EmployeeHours(UUID id, @NotNull String employeeId, @NotNull float contributionHours, float billableHours, float ptoHours,LocalDate updatedDate,float targetHours, LocalDate asOfDate) {
+    public EmployeeHours(UUID id, String employeeId, Float contributionHours, Float billableHours, Float ptoHours, LocalDate updatedDate, Float targetHours, LocalDate asOfDate) {
```

Comment:
Why are the `@NonNull`s matching the below constructor not on this one? Looks like these were intentionally taken off?

Reply:
Since I changed the datatypes of the floats, I just regenerated the whole constructor. I didn't realize it omitted the `@NotNull`s, thanks.